### PR TITLE
modernize (const, let, class, update dep, etc)

### PR DESCRIPTION
**Pull request by jimmywarting**
- Updated all dependencies
- Switched to const class, let
- Switched rimraf and mkdirp dep for native version with recursive options

### CODE REVIEW

**Comment by RyanZim** [comment]
This removes graceful-fs, curious what the reasoning was?

**Comment by jimmywarting** [comment]
Didn't see graceful-fs as needed... fewer dependencies. less to install, faster to start.

**Review by RyanZim — CHANGES_REQUESTED** [review]
Please remove `package-lock.json` and add it to `.gitignore` or turn it off in `.npmrc`; lockfiles are an anti-pattern for libraries.

**Comment by RyanZim** [comment]
Sorry for the slow review here; just got out of the hospital. I'm OK with removing graceful-fs.

File: package.json [hunk 1]
```diff
@@ -16,24 +16,22 @@
...
   },
   "homepage": "https://github.com/jprichardson/node-klaw#readme",
-  "dependencies": {
-    "graceful-fs": "^4.1.9"
-  },
   "devDependencies": {
-    "mkdirp": "^0.5.1",
-    "rimraf": "^2.4.3",
-    "standard": "^14.3.1",
+    "@types/node": "^16.6.1",
```
[the remark this answers is not in the document]
Comment (jimmywarting) [reply package.json:32]: removed it, wasn't really needed

**Comment by jimmywarting** [comment]
> Sorry for the slow review here; just got out of the hospital. I'm OK with removing graceful-fs.

Oh, hope you are better now. I removed the lock file and `@types/node`

**Review by RyanZim — APPROVED** [review]
LGTM; I'm working on getting the broken CI fixed up; will probably need to rebase this once I have that fixed.

**Comment by RyanZim** [comment]
Landed in 8fbf2d26a2b7f46dce7915052d25947ce6b9a0d2; thanks!

**Pull request closed without merging**